src/recipient.js

```diff
@@ -235,11 +254,19 @@ Recipient.prototype.processIndexedEarnings_ = function() {
   if (this.monthlyIndexedEarnings === 0 &&
       this.earningsRecords.length === 0)
     return;
-
+  
+  this.earnedCredits = 0;
+  this.plannedCredits = 0;
+  this.neededYears = -1;
+  this.lastNeededYear = 100000;
```

Comment:
This also doesn't appear to be used. I suspect you intended to use these two variables to give the user an idea of how many more years they needed to earn 40 credits, but that still would need to be finished. Just pointing out since I don't know what your intent was.

Reply:
It was in the UI at one point, but was removed.  Removed.